docs/user-guide/cli-using-daemon-mode.md

```diff
@@ -28,16 +28,29 @@ The following steps describe how to enable daemon mode and how to configure Zowe
    zowe daemon enable
    ```
 
-   The command copies the Zowe executables for your operating system into the $ZOWE_CLI_HOME/bin (.zowe/bin) directory. The next command that you issue starts the daemon.
+   The command copies the Zowe executables for your operating system into the `$ZOWE_CLI_HOME/bin` (`.zowe/bin`) directory. The next command that you issue starts the daemon.
 2. Add the path to the Zowe executables to the environment variables for your computer. For example:
 
    ```
    C:\Users\<user_ID>\.zowe\bin
    ```
+   **Alternative configuration**: By default, the daemon binary creates or reuses a file in the user's home directory each time a Zowe CLI command runs. In some cases, this behavior might be undesirable. For example, the home directory resides on a network drive and has poor file performance. To change the location that the daemon uses, set the following environment variable for your operating system:
+
+   - **Windows:** `ZOWE_DAEMON_LOCK`
+       
+       Specify an alternative path to the lock file that restricts access to the named pipe that the daemon uses for communication.
+       
+       **Default:**  `%HOMEPATH%\.zowe-daemon.lock`
+
+   - **Linux and macOS:** `ZOWE_DAEMON`
+      
+      Specify an alternative path to the socket that the daemon uses for communication.
+      
+      **Default:** `~/.zowe-daemon.sock`
```

Comment:
While this is technically correct, do we want to replace the `~` with `$HOME` to make it more consistent with the Windows default? It's not totally necessary, but it might make the documentation more consistent.